### DISCUSSION TITLE
After changing to bootstrap-email no data is showing

### DISCUSSION THREAD

**Discussion by RayNawara** [bootstrap-email/bootstrap-email#235] [Help]
Hi!

I am adding a mailer for orders. I have it working on my Rails 7 app but it's ugly because we use a lot of Bootstrap for formatting.

So I saw this gem and thought 'What a great and easy solution!'

I followed the Rails install but when I run I get the outline of a message but no data.

![bootstrap-email](https://github.com/bootstrap-email/bootstrap-email/assets/43546443/ec9a54ae-3557-4996-9baa-8f9a0b31fedb)

Any ideas? Are there any tutorials. I figure I must be missing something simple. Thanks in advance for any help!!

Ray

**Comment by stuyam** [comment]
Hi @RayNawara, could you past some example code so I can maybe get a better understanding of what is going wrong. Thank you
